### DISCUSSION TITLE
Blenderbot 2.0 pretrained weights

### DISCUSSION THREAD

**Discussion by rvong65** [facebookresearch/ParlAI#4055] [Q&A] [answered]
Where can I find the pretrained weights for Blenderbot 2.0 and how can I access them?

**Comment by klshuster** [comment, marked as answer]
BlenderBot 2.0 can be accessed via [ParlAI's model zoo](https://parl.ai/docs/zoo.html#wizard-of-internet-models); example invocations in the docs will download the relevant weights.
